channels/candidate-4.5.yaml

```diff
@@ -23,7 +23,25 @@ versions:
 - 4.4.10
 - 4.4.9
 - 4.4.8
+- 4.4.7
 - 4.4.6
+- 4.4.5
+- 4.4.4
+- 4.4.3
+- 4.4.2
+- 4.4.0
+- 4.4.0-rc.0
```

Comment:
I thought we were only adding versions from the previous minor if they had made it to stable channel?

Reply:
I'm only adding versions *to stable* from the previous minor if they were in the previous minor's stable channel.  I'm adding versions to candidate from the previous minor if they made it to the previous minor's candidate channel, etc.  This means that we're basically making one call about stability, and then putting the release in all 4.y and 4.(y+1) channels that match that stability decision.

Reply:
Yeah but we never tested upgrades from 4.4.0-rc.0 to any 4.5 version outside of immediately after branching which is usually a dumpster fire anyway.

Reply:
bah, nevermind, that doesn't mean we'd have edges, guess this is fine though confusing

Reply:
yeah, hopefully this PR adds no edges.  The idea is just to let folks say "I would like to be on $next_minor" regardless of which release they are on in the current minor, vs. today when they have to update to some threshold patch release before they can change channels.